sc_rf_service/sc_rf_service.py

```diff
@@ -1,24 +1,206 @@
-from asyncio import get_event_loop, sleep
+from asyncio import create_subprocess_exec, get_event_loop, sleep
 from datetime import datetime
 from random import random, randrange, uniform
+from typing import List
 
 from caproto import ChannelEnum, ChannelFloat, ChannelInteger, ChannelType
-from caproto.server import (PVGroup, PvpropertyBoolEnum, PvpropertyChar,
-                            PvpropertyEnum, PvpropertyEnumRO, PvpropertyFloat,
-                            PvpropertyFloatRO, PvpropertyInteger, PvpropertyDouble,
-                            PvpropertyString, ioc_arg_parser, pvproperty, run)
-from lcls_tools.superconducting.sc_linac_utils import (L1BHL, LINAC_TUPLES,
-                                                       ESTIMATED_MICROSTEPS_PER_HZ,
-                                                       PIEZO_HZ_PER_VOLT)
+from caproto.server import (
+    PVGroup,
+    PvpropertyBoolEnum,
+    PvpropertyChar,
+    PvpropertyDouble,
+    PvpropertyEnum,
+    PvpropertyEnumRO,
+    PvpropertyFloat,
+    PvpropertyFloatRO,
+    PvpropertyInteger,
+    PvpropertyString,
+    ioc_arg_parser,
+    pvproperty,
+    run,
+)
+from lcls_tools.superconducting.sc_linac_utils import (
+    ESTIMATED_MICROSTEPS_PER_HZ,
+    L1BHL,
+    LINAC_TUPLES,
+    PIEZO_HZ_PER_VOLT,
+)
 
 from simulacrum import Service
 
 
+class SeverityProp(pvproperty):
```

Comment:
Ooh, this will be useful! We have a bunch PVs with these enum_strings. Great idea to make a class to handle it!